app/repository/document.py

```diff
@@ -102,6 +102,7 @@ def _get_query(db: Session) -> Query:
 
 def _dto_to_family_document_dict(dto: DocumentCreateDTO) -> dict:
     return {
+        "import_id": dto.import_id if dto.import_id else "",
```

Comment:
This would be better to default to `None` indicating not set - as opposed to "" indicating it is set but invalid